### PR TITLE
added information on --cap-lints to deny-warnings antipattern

### PR DESCRIPTION
**Pull request by llogiq**
This is an easier way to counter `#[deny(warnings)]` than finding and removing or commenting out the annotation.

### CODE REVIEW

File: anti_patterns/deny-warnings.md [hunk 1]
```diff
@@ -35,7 +35,10 @@ before there was none.
 All this conspires to potentially break the build whenever something changes.
 
 Furthermore, crates that supply additional lints (e.g. [rust-clippy]) can no
-longer be used unless the annotation is removed.
+longer be used directly. Note that there is a way around this: With the
+`--cap-lints=warn` command line argument, all `deny` lint errors are turned into
+warnings (there may still be `forbid` lints that will stop compilation, but
+those are a different topic).
```
Comment (nrc) [inline anti_patterns/deny-warnings.md]: Either you should explain the difference between deny and forbid, or not mention forbid at all

**Comment by pickfire** [comment]
@llogiq ping

**Comment by simonsan** [comment]
@llogiq Could you please add an explanation for the difference between `deny` and `forbid` so we can merge? Thank you :-)

**Review by simonsan — APPROVED** [review]

File: anti_patterns/deny-warnings.md [hunk 2]
```diff
@@ -36,7 +36,10 @@ All this conspires to potentially break the build whenever something changes.
 
 Furthermore, crates that supply additional lints (e.g. [rust-clippy]) can no
 longer be used unless the annotation is removed. This is mitigated with
-[--cap-lints].
+[--cap-lints]. The `--cap-lints=warn` command line argument, turns all `deny`
+lint errors into warnings. But be aware that `forbid` lints are stronger than
+`deny` hence the 'forbid' level can not be overridden to be anything lower than
```
Comment (MarcoIeni) [inline anti_patterns/deny-warnings.md:41]:
```suggestion
`deny` hence the 'forbid' level cannot be overridden to be anything lower than
```

I am not native English speaker, so I don't know if I am right here.

Comment (simonsan) [reply anti_patterns/deny-warnings.md:41]: good spot

**Review by simonsan — APPROVED** [review]

**Pull request merged**